flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/FlinkUtils.java

```diff
@@ -207,7 +213,10 @@ public static void waitForClusterShutdown(
             if (!jobManagerRunning && !serviceRunning) {
                 break;
             }
-            LOG.info("Waiting for cluster shutdown... ({})", i);
+            // log a message waiting to shutdown Flink cluster every 5 seconds.
+            if (i % 4 == 0) {
```

Comment:
Wouldn't this actually print it every 4 seconds?

Reply:
@gyfora , i start from 0, therefore print the log every 5 seconds.

Reply:
I think that is actually incorrect, it will print:
1, 5, 9, 13...

Reply:
@gyfora, sorry for this mistake. I will update this as soon as possible.